docs/.vuepress/theme/styles/theme.styl

```diff
@@ -12,15 +11,16 @@ body, html
   overflow-x hidden
 
 body
-  font-family $defaultFont, -apple-system, "Helvetica Neue", sans-serif
+  font-family -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol"
```

Comment:
Let's use the `system-ui` family name as the 1st item here. See link from the issue description: https://furbo.org/2018/03/28/system-fonts-in-css/

Reply:
Gotcha. I used github's font stack but happy to go with the suggestion

Github:
`-apple-system,BlinkMacSystemFont,Segoe UI,Helvetica,Arial,sans-serif,Apple Color Emoji,Segoe UI Emoji`

Booking.com:
`BlinkMacSystemFont,-apple-system,Segoe UI,Roboto,Helvetica,Arial,sans-serif;`

It's worth noting that Medium uses `system` at the top of the stack though, rather than `system-ui`